### PR TITLE
syncthing: add discosrv binary and manpages

### PR DESCRIPTION
**Pull request by faulesocke**

### CODE REVIEW

File: srcpkgs/syncthing/template [hunk 1]
```diff
@@ -42,5 +43,23 @@ syncthing-relaysrv_package() {
 		vmove usr/bin/strelaysrv
 		vlicense cmd/strelaysrv/LICENSE
 		vsv relaysrv
+		vman man/strelaysrv.1
+	}
+}
+
+syncthing-discosrv_package() {
+	short_desc+=" - discovery server"
+	system_accounts="discosrv"
```
Comment (Duncaen) [inline srcpkgs/syncthing/template]: can we get this to `_discosrv` instead, `relaysrv` uses the old way because changing it is annoying.

Comment (faulesocke) [reply srcpkgs/syncthing/template]: Did that and also fixed the run-script.

File: srcpkgs/syncthing/template [hunk 2]
```diff
@@ -42,5 +43,23 @@ syncthing-relaysrv_package() {
...
+}
+
+syncthing-discosrv_package() {
+	short_desc+=" - discovery server"
+	system_accounts="_discosrv"
+	discosrv_homedir="/var/lib/discosrv"
+
+	make_dirs="
+	 /var/log/discosrv 700 root root
+	 /var/lib/discosrv 700 _discosrv _discosrv"
```
Comment (sgn) [inline srcpkgs/syncthing/template]: Why do we need to mkdir twice?

File: srcpkgs/syncthing/template [hunk 3]
```diff
@@ -42,5 +43,23 @@ syncthing-relaysrv_package() {
...
 		vlicense cmd/strelaysrv/LICENSE
 		vsv relaysrv
+		vman man/strelaysrv.1
+	}
+}
+
+syncthing-discosrv_package() {
+	short_desc+=" - discovery server"
+	system_accounts="_discosrv"
+	discosrv_homedir="/var/lib/discosrv"
```
Comment (sgn) [inline srcpkgs/syncthing/template]: `s/discos/_&`

File: srcpkgs/syncthing/template (hunk 2 above)
Comment (faulesocke) [reply srcpkgs/syncthing/template]: ? one is `log`, one is `lib`...

File: srcpkgs/syncthing/template (hunk 3 above)
Comment (faulesocke) [reply srcpkgs/syncthing/template]: Oh.

File: srcpkgs/syncthing/template (hunk 2 above)
Comment (sgn) [reply srcpkgs/syncthing/template]: Okay, I was sleepy

**Comment by faulesocke** [comment]
@Duncaen is this ready now or do I still need to fix something?

**Comment by Anachron** [comment]
Thanks!

**Pull request closed without merging**